src/testup/ui/html/preferences.html

```diff
@@ -58,11 +58,11 @@
       <su-group title="Code Editor" id="tuCodeEditor">
 
         <div class="tu-input-group">
-          <su-input label="Application:" v-model="config.editor.executable"></su-input>
+          <su-input label="Application:" v-on:keyup.native="changeEditorExecutable" v-model="config.editor.executable"></su-input>
```

Comment:
Hmm... this doesn't look right. `v-model` should be taking care of this - bending to `config.editor.executable` and updating it. `keyup` would not update if something was copied and pasted - while `v-model` should. We'd better figure out why `v-model` isn't properly updating.

Reply:
I am very sorry, when I modified this bug, I only considered solving the problem and forgot that this is open source collaboration. Thank you for your reminder.

Reply:
No problem - I know what its like when you just need to get something blocking you fixed.